src/main/resources/templates/chores/create.html

```diff
@@ -8,7 +8,7 @@
 <header th:replace="fragments :: header"></header>
 <form method="post" action="/chores/create">
     <div class="form-group">
-        <label>Name:
+        <label>Chore:
```

Comment:
```suggestion
        <label>Task:
```
We could call it "Task" to match the "TreasureTask" name!